lib/publiccloud/instance.pm

```diff
@@ -399,21 +430,6 @@ sub wait_for_ssh {
     my $instance_msg = "instance: $self->{instance_id}, public IP: $self->{public_ip}";
     my ($duration, $exit_code, $sshout, $sysout);
 
-    # skip SLES4SAP as incompatible with get_public_ip
-    if (!get_var('PUBLIC_CLOUD_SLES4SAP') && !$args{wait_stop}) {
```

Comment:
Kind question @pdostal: here you explicitly excluded the use of the routine when stopping instance: would it still be needed ?

Reply:
I don't think so. I don't call `update_instance_ip` before `wait_for_ssh(wait_stop => 1)`